Apps/Examples/Examples/All Examples/DebugMapExample.swift

```diff
@@ -0,0 +1,179 @@
+import UIKit
+import MapboxMaps
+
+private protocol DebugOptionSettingsDelegate: AnyObject {
+    func debugOptionSettingsDidChange(_ controller: SettingsViewController)
+}
+
+private struct MapDebugOptionSetting {
+    let debugOption: MapDebugOptions
+    let displayTitle: String
+}
+
+final class DebugMapExample: UIViewController, ExampleProtocol, DebugOptionSettingsDelegate {
+
+    private var map: MapboxMap!
```

Comment:
Almost everywhere we refer to this as mapboxMap instead of just map to avoid ambiguity with the `map` function and with things like [`Map`](https://developer.android.com/reference/java/util/Map) on android.

Reply:
Change suggested in https://github.com/mapbox/mapbox-maps-ios/pull/1225#discussion_r838886458 is made, and now we will access `debugOptions` with `mapView.mapboxMap.debugOptions`.